src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -36,10 +40,9 @@ export interface TemplateWrapperProps {
 const TemplateWrapper: React.FC<TemplateWrapperProps> = ({ venue }) => {
   const history = useHistory();
   const match = useRouteMatch();
+  const parentVenue = useSelector(parentVenueSelector);
```

Comment:
Where is the 'connect' for this query? And are we sure it will have been run before we try and read from it here?

Reply:
The connect for this query is actually in `NavBar` and only there. Which is not the greatest idea since screens without it won't have this data connected. I will back log this once I'm done fixing the comments.